Apps/MAUI.Demo/App.xaml.cs

```diff
@@ -106,15 +104,15 @@ protected override void OnStart()
 
     private string GetOneCollectorTokenString()
     {
-        return $"iosTarget={Environment.GetEnvironmentVariable("MAUI_IOS_TARGET_TOKEN_PROD")};" +
-            $"androidTarget={Environment.GetEnvironmentVariable("MAUI_ANDROID_TARGET_TOKEN_PROD")};" +
+        return "iosTarget={MAUI_IOS_TARGET_TOKEN_PROD};" +
+            "androidTarget={MAUI_ANDROID_TARGET_TOKEN_PROD};" +
             $"windowsTarget={Environment.GetEnvironmentVariable("MAUI_WINDOWS_TARGET_TOKEN_PROD")}";
```

Comment:
Does it work on such state?

Reply:
This works on windows. The variables are picked up from the system environment vars.